notebooks/how_to_handle_axes.ipynb

```diff
@@ -215,8 +209,8 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/edarin/.virtualenvs/openfisca/lib/python2.7/site-packages/ipykernel/__main__.py:1: RuntimeWarning: invalid value encountered in divide\n",
-      "  if __name__ == '__main__':\n"
+      "/usr/lib/python2.7/dist-packages/ipykernel_launcher.py:1: RuntimeWarning: invalid value encountered in divide\n",
```

Comment:
The error message might confuse readers

Reply:
Besides, how could we avoid this warning at execution?

Reply:
I you do not execcute the notebook it won't display useful information when you look at it on github